### DISCUSSION TITLE
Adding Variable in Workshop Console fails unless it is a Secret

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#108] [Bugs] [answered]
Version: 0.6.0

![Screenshot 2024-02-08 at 10 38 37 AM](https://github.com/noop-inc/documentation/assets/3111670/79ec3911-7bba-4a7d-bb5e-bbd2f4f514d1)

**Comment by derekwolpert** [comment, marked as answer]
Thanks. This bug will be resolved in the next release of Noop Desktop.
